### DISCUSSION TITLE
Initial MicroPython custom board definition

### DISCUSSION THREAD

**Discussion by andypiper** [andypiper/fivebyfive#9] [Announcements]
I've added an initial attempt at a board definition:

https://github.com/andypiper/fivebyfive/tree/main/micropython/board

If you're familiar with MicroPython firmware builds, let me know what you think!

**Comment by ozett** [comment]
make a documentation how to flash.
may you consider putting circuitpython on the C3-Board?
for circuitpython there are lots of C3 and S2 boards. 
maybe there is more troubleshooting information for this board-types there

**Comment by andypiper** [comment]
Documentation on how to build and flash this definition is a great idea, I will look into that, thanks.

I did talk to the CircuitPython folks about the C3 board a while ago and there was no way at the time to load CPy code onto them - they were looking at using BLE for it, but I don’t know if that was ever completed. I will take another look.
